package.json

```diff
@@ -17,7 +17,6 @@
     "build": "yarn build:lib && yarn build:index && yarn build:fixtures && yarn build:remote-schemas",
     "ci": "yarn test",
     "dev": "DEBUG=info,warn,error nf start --procfile Procfile.dev -w",
-    "dump-remote-schema": "babel-node --extensions '.ts,.js' ./scripts/dump-remote-schema.ts",
     "dump-schema": "babel-node --extensions '.ts,.js' ./scripts/dump-schema.ts",
```

Comment:
@jonallured - mind adding a `"sync-schema:convection": "copy ../_schema.graphql data/convection.graphql"` command here?

Reply:
Might be overkill - we have that command documented over in convection and it makes more sense to me to copy from there rather than from MP. 🤷‍♂